src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/DetailsViewModelMapper.cs

```diff
@@ -98,8 +99,11 @@ private void PopulateFundingBandExcessModels(List<DetailsViewCourseGroupingModel
                 if (numberExceedingBand > 0)
                 {
                     var fundingExceededValues = apprenticesExceedingFundingBand.GroupBy(x => x.FundingBandCap).Select(fundingBand => fundingBand.Key);
+                    var fundingBandCapExcessHeader = SetFundingBandExcessHeader(apprenticesExceedingFundingBand.Count);
```

Comment:
Just a personal preference, instead of Set... can it be Get.... . As we are getting a value.